File: android/app/src/main/kotlin/com/bitchat/mesh/dec_chat/ble/BleConstants.kt
package com.bitchat.mesh.dec_chat.ble

import java.util.UUID

object BleConstants {
    val SERVICE_UUID: UUID = UUID.fromString("0000FDC7-0000-1000-8000-00805F9B34FB")
    val PACKET_CHARACTERISTIC_UUID: UUID = UUID.fromString("00002A06-0000-1000-8000-00805F9B34FB")

    const val CONTROL_CHANNEL_NAME = "com.bitchat.mesh/ble_control"
    const val EVENT_CHANNEL_NAME = "com.bitchat.mesh/ble_events"

    const val TARGET_MTU = 512
}

File: ios/Runner/BLE/BLEConstants.swift
import Foundation
import CoreBluetooth

enum BLEConstants {
    static let serviceUUID = CBUUID(string: "0000FDC7-0000-1000-8000-00805F9B34FB")
    static let packetCharacteristicUUID = CBUUID(string: "00002A06-0000-1000-8000-00805F9B34FB")
    
    static let controlChannelName = "com.bitchat.mesh/ble_control"
    static let eventChannelName = "com.bitchat.mesh/ble_events"
    
    static let targetMTU: Int = 512
}
